src/style.css

```diff
@@ -1 +1,34 @@
 /* add styles here */
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
it is not a good idea to use * selector for resetting margins and padding. Use it only for box-sizing property